os-eap-logging/added/logging.properties

```diff
@@ -32,7 +32,7 @@ handlers=CONSOLE
 
 handler.CONSOLE=org.jboss.logmanager.handlers.ConsoleHandler
 handler.CONSOLE.level=ALL
-handler.CONSOLE.formatter=COLOR-PATTERN
+handler.CONSOLE.formatter=##CONSOLE-FORMATTER##
```

Comment:
Having this disconnected from the related script is a problem, as this module is used in other images, and those images might not use os-eap7-launch. For sure EAP 6.4 doesn't, probably other products too.

Reply:
BTW I don't mean to suggest this file must move to be with the script. I'm not suggesting a solution, just a pointing out a problem. :)

A possible solution is to limit this fix to EAP CD and EAP on OS 7.2 which can use the module from jboss-eap-modules.

Reply:
Yeah, my intention there was to do the substitution if it was there, and if it wasn't it would just be a no-op. Other images can add the ##CONSOLE-FORMATTER## as they see fit.

I'll have a quick look in cct_module and see if there are any other examples out there, then ping those people with a heads up.

Reply:
Actually never mind, sorry, I see what you mean about the file being in os-eap-logging, but the substitution being in os-eap7-launch, I'll have a look at what we can do in os-eap-logging ...

Reply:
The problem is the logging.properties file is no longer usable as is because the handler.CONSOLE.formatter=##CONSOLE-FORMATTER## is invalid.

Reply:
@bstansberry Yeah, got it, theres a stuck pending comment from me just above your last message :(

Reply:
Sigh, the GH UI thought I'd started a review ...

Reply:
So thinking about this a little more, we should fork the cct_module os-eap-logging module to jboss-eap-modules, and install the script that does the substitution into launch/ and ensure it gets run. We can deprecate the old one, but it would still work.